enterprise/api/renderers.py

```diff
@@ -8,13 +8,18 @@
 from rest_framework.settings import api_settings
 
 
-class IgnoreClientContentNegotiation(DefaultContentNegotiation):
+class CustomContentNegotiation(DefaultContentNegotiation):
     """
-    Class for client content negotiation.
+    DRF was ignoring the order in render_classes, may be due to djangorestframework-xml library.
+    By Default, It was giving precedence to xml by default so we need to override the select_renderer method.1
```

Comment:
nit: `By Default, It was giving precedence to xml by default so we need to override the select_renderer method.1` -> `By Default, It was giving precedence to XML so we need to override the select_renderer method`

Reply:
Was it giving you XML even when you specified the appropriate headers `Accept: application/json`?

Reply:
Can we make an upstream PR to allow the default content type returned to be configurable?

Reply:
Seems good but creating a new ticket would be helpful.